src/Adapter/Http/HttpConnector.php

```diff
@@ -127,7 +136,7 @@ private function request($method, $path, $body, array $options)
             if ($response = $e->getResponse()) {
                 throw $this->createRequestException($e);
             }
-
+            $this->logger->error(get_class($e).' - '.$e->getMessage().' - '.$e->getTraceAsString());
```

Comment:
Can you use the exception in the `exception` key of the context instead, so we can see it "properly" in Sentry? Also, it's a `warning` level I believe, as it's not a CP-problem per say I believe (at least most of the time as the cluster is a user cluster)